src/main/java/io/fabric8/vertx/maven/plugin/mojos/Constants.java

```diff
@@ -124,4 +124,21 @@
     String VERTX_RUN_MODE_JAR = "jar";
 
 
+    /**
+     *
+     */
+    String WILDCARD_JSON_FILES = "*.json";
+    /**
+     *
+     */
+    String WILDCARD_YAML_FILES = "*.yaml";
+    /**
+     *
+     */
+    String WILDCARD_YML_FILES = "*.yml";
```

Comment:
Thanks for letting me know that.  I thought the other way thinking thats the better practice .. will have this in mind now.